x/staking/spec/begin_redelegation_sequence.puml

```diff
@@ -0,0 +1,51 @@
+@startuml
```

Comment:
Where are these diagrams displayed?

Ideally, we could display them in https://docs.cosmos.network/v0.41/modules/staking/, or in a subsection there. Would it be possible to include the diagram in a .md file? (we use vuepress's md flavor, if that helps).